src/main/java/net/mcreator/workspace/elements/VariableType.java

```diff
@@ -40,10 +45,28 @@ public String getName() {
 		return name;
 	}
 
-	public int getColor() {
+	public String getColor() {
 		return color;
 	}
 
+	/**
+	 * Returns the color of the blocks associated with this variable type. If the field is a valid hex color code, it's
+	 * returned as-is. If it's a valid integer, it's treated as a hue to get the color with the correct saturation and
+	 * value.
+	 * @return The color of the associated blocks, or black if it's badly formatted.
+	 */
+	public Color getBlocklyColor() {
+		try {
+			if (!color.startsWith("#"))
+				return BlocklyBlockUtil.getBlockColorFromHUE(Integer.parseInt(color));

```

Comment:
I think this line is not formatted. The spacing ia strange.

Reply:
Not sure where the issue is, the spacing is fine to me

Reply:
Ok, it's maybe due to my phone.